DOCS/Users_Guide/CMAQ_UG_ch13_WRF-CMAQ.md

```diff
@@ -32,41 +32,12 @@ Hemispheric WRF-CMAQ model simulation over two decades (1990−2010) shows enhan
 
 ## 13.4 Latest WRF-CMAQ Release
 
-The new WRF-CMAQ model is based on WRFv4.3 and CMAQv5.3.3. It supports only RRTMG radiation scheme for short wave aerosol direct effect. It uses core-shell model to perform aerosol optics calculations rather than volume mixing technique as in the previous version of the WRF-CMAQ model. 
+The new WRF-CMAQ model is based on WRFv4.4 and CMAQv5.3.3 and later. It supports only RRTMG radiation scheme for short wave aerosol direct effect. It uses core-shell model to perform aerosol optics calculations rather than volume mixing technique as in the previous version of the WRF-CMAQ model. 
```

Comment:
@kmfoley be careful with this statement. I think it should read "WRFv4.4 and CMAQ5.3.3+ and later. I'm not sure v5.3.3 or any earlier versions will work with WRFv4.4